app/src/pages/playground/PlaygroundRunButton.tsx

```diff
@@ -1,28 +1,52 @@
 import React from "react";
+import { css } from "@emotion/react";
 
 import { Button, Icon, Icons } from "@arizeai/components";
 
+import { Loading } from "@phoenix/components";
 import { usePlaygroundContext } from "@phoenix/contexts/PlaygroundContext";
 
 export function PlaygroundRunButton() {
   const runPlaygroundInstances = usePlaygroundContext(
     (state) => state.runPlaygroundInstances
   );
+  const cancelPlaygroundInstances = usePlaygroundContext(
+    (state) => state.cancelPlaygroundInstances
+  );
   const isRunning = usePlaygroundContext((state) =>
     state.instances.some((instance) => instance.activeRunId != null)
   );
   return (
     <Button
       variant="primary"
-      disabled={isRunning}
-      icon={<Icon svg={<Icons.PlayCircleOutline />} />}
-      loading={isRunning}
+      icon={
+        !isRunning ? (
+          <Icon svg={<Icons.PlayCircleOutline />} />
+        ) : (
+          <div
+            css={css`
+              margin-right: var(--ac-global-dimension-static-size-50);
+              & > * {
+                height: 1em;
+                width: 1em;
+                font-size: 1.3rem;
+              }
+            `}
+          >
+            <Loading size="S" />
+          </div>
+        )
+      }
```

Comment:
we  have a loading prop on our buttons you can use instead of this i think

Reply:
It also disables the button, so I can't use it

Reply:
ahh okay, this works then